docs/source/researchers-start-here.rst

```diff
@@ -22,88 +21,112 @@ practices, etc. It’s also where you should go with any questions or
 technical support requests. There are a lot of participating researchers
 (over 400 at last count!) who may be able to help!
```

Comment:
Is 'over 400' still correct for the number of researcher accounts, or is it more now?